azavea/react-showtime
# Create workflow to publish to npm

## Overview

Establishes a GitHub Actions workflow to build and publish `react-showtime` to npm.

Resolves #4 

## Testing Instructions

- Verify that the [Netlify build](https://app.netlify.com/sites/react-showtime/deploys/5fb7eed9c976910007ab32cf) passes.
- Verify that the [deploy preview](https://deploy-preview-7--react-showtime.netlify.app/) is available with no content security policy errors in the console.
- Verify that the [CI workflow passes](https://github.com/azavea/react-showtime/pull/7/checks?check_run_id=1432080802), including all linting and tests.
- Confirm that all four files in `dist` will be included in the final released package as part of the ["Build and publish package" step in the Release workflow](https://github.com/azavea/react-showtime/pull/7/checks?check_run_id=1432080568).
